Misc/ACKS

```diff
@@ -1889,16 +1901,3 @@ Jelle Zijlstra
 Gennadiy Zlobin
 Doug Zongker
 Peter Åstrand
-Zheao Li
-Carsten Klein
-Diego Rojas
-Edison Abahurire
-Geoff Shannon
-Batuhan Taskaya
-Aleksandr Balezin
-Robert Leenders
-Tim Hopper
-Dan Lidral-Porter
-Ngalim Siregar
-Tim Gates
-Karthikeyan Singaravelan
```

Comment:
12 lines added, 13 lines removed because Ngalim Siregar was already in the right spot.